test/unit/parse_run_test.rb

```diff
@@ -230,6 +230,120 @@ class ParseRunTest < Minitest::Test
       assert result.status
     end
 
+    test "#check does not list stale violations when run on a single file with SOME violations, even if the containing package has violations" do
```

Comment:
```suggestion
    test "#check does not list stale violations when run on a single file with multiple violations, even if the containing package has violations" do
```
Or do you mean a violation from a different package? I'm not sure I'm understanding the test correctly.

Reply:
Basically the file that we run on has violations in this test, versus in the other test the file we passed in didn't have any violations. I can just remove `SOME` entirely I think.